specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerAppsBuilds.json

```diff
@@ -0,0 +1,403 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsBuildNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppsBuildResource"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ContainerAppsBuilds_Get_0": {
+            "$ref": "./examples/ContainerAppsBuilds_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ContainerAppsBuilds"
+        ],
+        "operationId": "ContainerAppsBuilds_Delete",
+        "description": "Delete a ContainerAppsBuild resource",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsBuildNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Resource deletion accepted.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Resource deleted successfully."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ContainerAppsBuilds_Delete_0": {
+            "$ref": "./examples/ContainerAppsBuilds_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "ContainerAppsBuildConfiguration": {
+      "type": "object",
+      "properties": {
+        "baseOs": {
```

Comment:
If builds are created implicitly then should most of these properties be marked as read-only?

Reply:
Yes, especially if they're intended to never be updatable.